static/faq.html

```diff
@@ -0,0 +1,111 @@
+        <meta property="og:description" content="Frequently asked questions about GrapheneOS."/>
+        <meta property="og:type" content="website"/>
+        <meta property="og:image" content="https://grapheneos.org/opengraph.png"/>
+        <meta property="og:image:width" content="512"/>
+        <meta property="og:image:height" content="512"/>
+        <meta property="og:image:alt" content="GrapheneOS logo"/>
+        <meta property="og:url" content="https://grapheneos.org/faq"/>
+        <meta property="og:site_name" content="GrapheneOS"/>
+        <link rel="icon" type="image/vnd.microsoft.icon" href="/favicon.ico"/>
+        <link rel="mask-icon" href="/safari_pinned_tab_icon.svg" color="#000000"/>
+        <link rel="stylesheet" href="/grapheneos.css?13"/>
+        <link rel="manifest" href="/manifest.webmanifest"/>
+        <link rel="canonical" href="https://grapheneos.org/faq"/>
+    </head>
+    <body>
+        <nav>
+            <ul>
+                <li><a href="/">GrapheneOS</a></li>
+                <li><a href="/install">Install</a></li>
+                <li><a href="/build">Build</a></li>
+                <li><a href="/usage">Usage</a></li>
+                <li><a href="/releases">Releases</a></li>
+                <li><a href="/source">Source</a></li>
+                <li><a href="/donate">Donate</a></li>
+                <li><a href="/contact">Contact</a></li>
+            </ul>
+        </nav>
+        <div id="content">
+<!--Site header above. Do not edit above this line.-->        
+            <h1 id="faq">
+                <a href="#faq">Frequently Asked Questions</a>
+            </h1>
+            
+            <h2 id="table-of-contents">
+                <a href="#table-of-contents">Table of contents</a>
+            </h2>
+<!--Stub Entry
+            <ul>
+                <li>
+                    <a href="#about-grapheneos">About GrapheneOS</a>
+                    <ul>
+                        <li>[tba]</li>
+                    </ul>
+                </li>    
+                <li>
+                    <a href="#installing-grapheneos">Installation</a>
+                    <ul>
+                        <li>[tba]</li>
+                    </ul>
+                </li>
+                    <a href="#security-in-grapheneos">Security in GrapheneOS</a>
+                    <ul>
+                        <li>[tba]</li>
+                    </ul>
+                </li>
+            </ul>
+-->
+
+<!--Content below -->
+
+<!--Site footer below. Do not edit below this line.--> 
+        </div>
+        <footer>
+            <a href="/"><img src="https://grapheneos.org/logo.png" width="512" height="512" alt=""/>GrapheneOS</a>
+            <ul id="social">
+                <li><a href="https://twitter.com/GrapheneOS">Twitter</a></li>
+                <li><a href="https://github.com/GrapheneOS">GitHub</a></li>
+            </ul>
+        </footer>
+        <script type="application/ld+json">
```

Comment:
Can you leave out this application/ld+json part? It's only meant for the contact page.

Reply:
Whoops, sure. Pushing it now.